encoding/av1-break-even-calculator/index.html

```diff
@@ -0,0 +1,183 @@
+          <small>&nbsp;</small>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="egressCostPerGb">CDN delivery cost per GB</label>
+        <div class="input-group">
+          <input class="form-control"
+                id="egressCostPerGb"
+                min="0"
+                step="0.01"
+                type="number"
+                value="0.04" />
+          <div class="input-group-append"><span class="input-group-text">$</span></div>  
+        </div>
+      </div>
+    </div>
+
+    <div class="col-md-6">
+      
+      <div class="form-group">
+        <label for="numberOfStreamsUhd">Amount of UHD streams</label>
+        <input class="form-control"
+              id="numberOfStreamsUhd"
+              min="0"
+              step="1"
+              type="number"
+              value="2" />
+        <div>
+          <small>&nbsp;</small>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="numberOfStreamsHd">Amount of HD streams</label>
+        <input class="form-control"
+              id="numberOfStreamsHd"
+              min="0"
+              step="1"
+              type="number"
+              value="3" />
+        <div>
+          <small>&nbsp;</small>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="numberOfStreamsSd">Amount of SD streams</label>
+        <input class="form-control"
+              id="numberOfStreamsSd"
+              min="0"
+              step="1"
+              type="number"
+              value="4" />
+      </div>
+    </div>
+
+  </div>
+</form>
+
+<div class="container">
+  <div class="row">
+
+    <div class="col-md-6 my-2">
+      <div>Break-even point - AV1 vs H.264</div>
+      <div id="av1-h264" class="font-weight-bold" style="font-size: 24px">--</div>
+    </div>
+     
+    <div class="col-md-6 my-2">
+      <div>Break-even point - AV1 vs H.265</div>
+      <div id="av1-h265" class="font-weight-bold" style="font-size: 24px">--</div>
+    </div>
+
+  </div>
+</div>
+<p class="text-justify">
+  Starting from these numbers of views, supplementing encodings with AV1 becomes more
+  profitable than the respective codec alone. 
+</p>
+
+<h3 class="my-3">Calculation multipliers</h3>
+<p class="text-justify">
+  These are the multipliers and factors used in our calculation
+  <a href="#formulaFootnote" target="_self" class="btm-footnote"></a>,
+  based on encoding minutes in pricing. 
+</p>
+<div class="container">
+  <div class="row">
+    <table class="table">
+      <thead>
+        <th colspan="3">Multipliers</th>
+      </thead>
+      <tbody>
+        <tr class="table-secondary">
+          <th class="col-3">SD</th>
+          <th class="col-3">HD</th>
+          <th class="col-3">UHD</th>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2</td>
+          <td>4</td>
+        </tr>
+        <tr class="table-secondary">
+          <th>PerTitle</th>
+          <th>2-Pass</th>
```

Comment:
2-pass factor is not used in the calculation
should be removed

Reply:
👍

Reply:
Removed in 2baee14